src/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans&display=swap"
+      rel="stylesheet"
+    />
+    <title>Stanley | To Do List</title>
+  </head>
+  <body>
+    <main>
+      <section id="main-content">
+        <header class="todo-header">
+          <h1>Today's To Do</h1>
+          <button class="arrow-rotate">
+            <i class="fa-solid fa-arrows-rotate"></i>
+          </button>
+        </header>
+
+        <form id="todo-form">
+          <input
+            type="text"
+            id="list-input"
+            name="list-input"
+            placeholder="Add your list..."
```

Comment:
- [ ] Great implementation of adding tasks to the task lists however, upon submit on no input, an empty task is added to the list.

Kindly note that the list should not have an empty task as that does not give a very good user experience. ⛔ 

A simple suggestion is to add 'required' attribute to the input element so that it does not submit on no input. 🧰

Reply:
@frempongdev  Great suggestion. Thank you.